src/libstd/collections/hash/map.rs

```diff
@@ -2439,6 +2457,17 @@ where
     fn extend<T: IntoIterator<Item = (&'a K, &'a V)>>(&mut self, iter: T) {
         self.base.extend(iter)
     }
+
+    #[inline]
+    fn extend_one(&mut self, (&k, &v): (&'a K, &'a V)) {
+        self.base.insert(k, v);
+    }
+
+    #[inline]
+    fn extend_reserve(&mut self, additional: usize) {
+        // self.base.extend_reserve(additional);
```

Comment:
nit: stray comment?

Reply:
Oh, that's sort of tied to the same FIXME above, but I'll just remove it. The forwarding here will still work fine once hashbrown takes over `extend_reserve` logic.